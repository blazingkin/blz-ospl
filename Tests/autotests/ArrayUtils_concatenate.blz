:main
    #Test 00 :concatenate
    print(">>> Test 00 :concatenate <<<")
    arr1 = [0,1,2]
    arr2 = [3,{nil},5,6,7]
    arr3 = arr1.concatenate(arr2)
    print("arr3: " + arr3)
    print("arr1: " + arr1)
    print("--------------------------------")
end