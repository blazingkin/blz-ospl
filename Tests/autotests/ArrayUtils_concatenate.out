>>> Test 00 :concatenate <<<
arr3: [0, 1, 2, 3, , 5, 6, 7]
arr1: [0, 1, 2]
--------------------------------
>>> Test 01 :concatenate! <<<
arr3: [0, 1, 2, 3, , 5, 6, 7]
arr1: [0, 1, 2, 3, , 5, 6, 7]
--------------------------------
